File: Linguagem C Exercicios/Vetores/NEGATIVOS.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int n,x;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d",&n);

    int vet[n];

    for (int i = 0;i< n;i++){
        printf("Digite um numero: ");
        scanf("%d",&vet[i]);
    };

    printf("NUMEROS NEGATIVOS:\n");

    for(int i=0;i<n;i++){
        if (vet[i] % 2 != 0){
            printf("%d\n",vet[i]);
        };
    };

    return 0;
}

File: Linguagem C#/Vetores/Negativos.cs
using System;

namespace Negativos {
    class Program {
        static void Main(string[] args) {

            int n;

            Console.Write("Quantos numeros voce vai digitar? ");
            n = int.Parse(Console.ReadLine());

            int[] vet = new int[n];

            for (int i = 0; i < n; i++) {
                Console.Write("Digite um numero: ");
                vet[i] = int.Parse(Console.ReadLine());
            };

            Console.WriteLine("NUMEROS NEGATIVOS:");
            for (int i = 0; i < n; i++) {
                if (vet[i] < 0) {
                    Console.WriteLine(vet[i]);
                };
            };
        }
    }
}
